src/node_api.cc

```diff
@@ -220,9 +222,9 @@ class ThreadSafeFunction : public node::AsyncResource {
 
     if (uv_async_init(loop, &async, AsyncCb) == 0) {
       if (max_queue_size > 0) {
-        cond.reset(new node::ConditionVariable);
+        cond = std::make_unique<node::ConditionVariable>();
       }
-      if ((max_queue_size == 0 || cond.get() != nullptr) &&
+      if ((max_queue_size == 0 || cond != nullptr) &&
```

Comment:
Btw, this works more by accident than by design: it converts both `cond` and `nullptr` to boolean before comparing them. I’d just write `... || cond) && ...`, personally.

Reply:
done.